Sources/eppo/EppoClient.swift

```diff
@@ -237,16 +237,16 @@ public class EppoClient {
         
         // Optionally log assignment
         if flagEvaluation.doLog {
-            if let allocationKey = flagEvaluation.allocationKey,
-               let variation = flagEvaluation.variation,
```

Comment:
Previously this conditional evaluated to `false` and did not fall through if either `flagEvaluation.allocationKey` or `flagEvaluation.variation` where `.none`. However to mirror the implementation in JS this relaxes the constraints and allows us to log, hypothetically, the sentinel null values.

Reply:
interesting